dwh/pyproject.toml

```diff
@@ -0,0 +1,2 @@
+[tool.black]
```

Comment:
How is this file is being used and how the `black` tool is automatically run on the code?

Reply:
Currently, I am running `black` tool from command line. When the `black` tool is run from command line or as part of IDE setting, the `black` tool looks for a `pyproject.toml` file and uses the settings specified in this file.

Later in another PR, I will add command line invocation for the `black` tool to make it easier. We can also add a check as a pre-commit condition. I will address that in the other PR I am working on.